### ISSUE TITLE
cli mode PHP has no default configuration file

### ISSUE THREAD

**Issue by ericsagnes** [NixOS/nixpkgs#9625]
Hi,

cli mode PHP has no default configuration file set:

~~~~
nix-shell -p php --pure
php --ini
Configuration File (php.ini) Path: /nix/store/fmwdjsmrkx9zlcf950mxhd9wkhi9wc3c-php-5.6.12/etc
Loaded Configuration File:         (none)
Scan for additional .ini files in: /etc
Additional .ini files parsed:      (none)
~~~~

Setting a [PHPRC](http://php.net/manual/en/configuration.file.php) environment variable to `php-recommended.ini` could be a solution.

**Comment by ericsagnes** [comment]
cc package maintainer @globin

**Comment by dsferruzza** [comment]
Hi!

I had the same problem and solved it by adding a `/etc/php.ini` file with my custom config.

```nix
environment.etc."php.ini".text = ''
   log_error = On
  ; whatever
'';
```

PHP CLI finds it and uses it!

```
# php --ini
Configuration File (php.ini) Path: /nix/store/6fr66njps4vwrpfyq0scg5s4y0qq25xs-php-5.6.20/etc
Loaded Configuration File:         (none)
Scan for additional .ini files in: /etc
Additional .ini files parsed:      /etc/php.ini
```

It is not perfect, but it seems to work :)

**Comment by ericsagnes** [comment]
fixed by #15018
